src/Marten.AsyncDaemon.Testing/event_projections_end_to_end.cs

```diff
@@ -171,7 +171,7 @@ public override void Apply(IDocumentOperations operations, IReadOnlyList<StreamA
                 switch (@event.Data)
                 {
                     case IEvent<Travel> e:
-                        var travel = e.GetData();
```

Comment:
You have to use the method for co/contravariance on the event wrappers. You can't make this change, or it'll break other functionality.